### ISSUE TITLE
Fresh emacs 29 + spacemacs fails to start

### ISSUE THREAD

**Issue by joakim-hove** [syl20bnr/spacemacs#16124]
I have compiled emacs29 on ubuntu - all defaults. I then clone a fresh spacemacs distribution, but emacs/spacemacs fails to start. The warning window shows:

```
Warning (initialization): An error occurred while loading ‘$HOME/.emacs.d/init.el’:

File is missing: Cannot open load file, No such file or directory, bind-map

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the ‘--debug-init’ option to view a complete error backtrace.
```

The Spacemacs welcome screen is shown in the top window, but it has clearly not been fully initialized and emacs is in a unusable state. When starting emacs with `--debug-init` I get:
```
Found 112 new package(s) to install...
--> refreshing package archive: nongnu... [3/3]
--> installing package: which-key@spacemacs-bootstrap... [1/112]
Package which-key is unavailable. Is the package name misspelled?
--> installing package: pcre2el@spacemacs-bootstrap... [2/112]
Package pcre2el is unavailable. Is the package name misspelled?
--> installing package: hydra@spacemacs-bootstrap... [3/112]
Package hydra is unavailable. Is the package name misspelled?
--> installing package: dotenv-mode@spacemacs-bootstrap... [4/112]
Package dotenv-mode is unavailable. Is the package name misspelled?
--> installing package: diminish@spacemacs-bootstrap... [5/112]
Package diminish is unavailable. Is the package name misspelled?
--> installing package: bind-map@spacemacs-bootstrap... [6/112]
Package bind-map is unavailable. Is the package name misspelled?
--> installing package: async@spacemacs-bootstrap... [7/112]
Package async is unavailable. Is the package name misspelled?
--> installing package: ws-butler@spacemacs-editing... [8/112]
Package ws-butler is unavailable. Is the package name misspelled?
--> installing package: writeroom-mode@spacemacs-editing-visual... [9/112]
Package writeroom-mode is unavailable. Is the package name misspelled?
--> installing package: winum@spacemacs-navigation... [10/112]
Package winum is unavailable. Is the package name misspelled?
--> installing package: window-purpose@spacemacs-purpose... [11/112]
Package window-purpose is unavailable. Is the package name misspelled?
--> installing package: volatile-highlights@spacemacs-editing-visual... [12/112]
Package volatile-highlights is unavailable. Is the package name misspelled?
--> installing package: vi-tilde-fringe@spacemacs-evil... [13/112]
Package vi-tilde-fringe is unavailable. Is the package name misspelled?
--> installing package: uuidgen@spacemacs-editing... [14/112]
Package uuidgen is unavailable. Is the package name misspelled?
--> installing package: undo-tree@spacemacs-editing... [15/112]
Package undo-tree is unavailable. Is the package name misspelled?
--> installing package: treemacs-projectile@treemacs... [16/112]
Package treemacs-projectile is unavailable. Is the package name misspelled?
--> installing package: treemacs-persp@treemacs... [17/112]
Package treemacs-persp is unavailable. Is the package name misspelled?
--> installing package: treemacs-icons-dired@treemacs... [18/112]
Package treemacs-icons-dired is unavailable. Is the package name misspelled?
--> installing package: treemacs-evil@treemacs... [19/112]
Package treemacs-evil is unavailable. Is the package name misspelled?
--> installing package: treemacs@treemacs... [20/112]
Package treemacs is unavailable. Is the package name misspelled?
--> installing package: toc-org@spacemacs-org... [21/112]
Package toc-org is unavailable. Is the package name misspelled?
--> installing package: symon@spacemacs-modeline... [22/112]
Package symon is unavailable. Is the package name misspelled?
--> installing package: symbol-overlay@spacemacs-navigation... [23/112]
Package symbol-overlay is unavailable. Is the package name misspelled?
--> installing package: string-inflection@spacemacs-editing... [24/112]
Package string-inflection is unavailable. Is the package name misspelled?
--> installing package: string-edit-at-point@spacemacs-editing... [25/112]
Package string-edit-at-point is unavailable. Is the package name misspelled?
--> installing package: spaceline@spacemacs-modeline... [26/112]
Package spaceline is unavailable. Is the package name misspelled?
--> installing package: smartparens@spacemacs-editing... [27/112]
Package smartparens is unavailable. Is the package name misspelled?
--> installing package: restart-emacs@spacemacs-navigation... [28/112]
Package restart-emacs is unavailable. Is the package name misspelled?
--> installing package: request@spacemacs-misc... [29/112]
Package request is unavailable. Is the package name misspelled?
--> installing package: rainbow-delimiters@spacemacs-editing-visual... [30/112]
Package rainbow-delimiters is unavailable. Is the package name misspelled?
--> installing package: quickrun@spacemacs-defaults... [31/112]
Package quickrun is unavailable. Is the package name misspelled?
--> installing package: projectile@spacemacs-project... [32/112]
Package projectile is unavailable. Is the package name misspelled?
--> installing package: posframe@spacemacs-visual... [33/112]
Package posframe is unavailable. Is the package name misspelled?
--> installing package: popwin@spacemacs-visual... [34/112]
Package popwin is unavailable. Is the package name misspelled?
--> installing package: popup@spacemacs-visual... [35/112]
Package popup is unavailable. Is the package name misspelled?
--> installing package: persp-mode@spacemacs-layouts... [36/112]
Package persp-mode is unavailable. Is the package name misspelled?
--> installing package: password-generator@spacemacs-editing... [37/112]
Package password-generator is unavailable. Is the package name misspelled?
--> installing package: paradox@spacemacs-navigation... [38/112]
Package paradox is unavailable. Is the package name misspelled?
--> installing package: overseer@emacs-lisp... [39/112]
Package overseer is unavailable. Is the package name misspelled?
--> installing package: org-superstar@spacemacs-org... [40/112]
Package org-superstar is unavailable. Is the package name misspelled?
--> installing package: open-junk-file@spacemacs-navigation... [41/112]
Package open-junk-file is unavailable. Is the package name misspelled?
--> installing package: nameless@emacs-lisp... [42/112]
Package nameless is unavailable. Is the package name misspelled?
--> installing package: multi-line@spacemacs-editing... [43/112]
Package multi-line is unavailable. Is the package name misspelled?
--> installing package: macrostep@emacs-lisp... [44/112]
Package macrostep is unavailable. Is the package name misspelled?
--> installing package: lorem-ipsum@spacemacs-editing... [45/112]
Package lorem-ipsum is unavailable. Is the package name misspelled?
--> installing package: link-hint@spacemacs-editing... [46/112]
Package link-hint is unavailable. Is the package name misspelled?
--> installing package: indent-guide@spacemacs-editing-visual... [47/112]
Package indent-guide is unavailable. Is the package name misspelled?
--> installing package: hungry-delete@spacemacs-editing... [48/112]
Package hungry-delete is unavailable. Is the package name misspelled?
--> installing package: hl-todo@spacemacs-visual... [49/112]
Package hl-todo is unavailable. Is the package name misspelled?
--> installing package: highlight-parentheses@spacemacs-editing-visual... [50/112]
Package highlight-parentheses is unavailable. Is the package name misspelled?
--> installing package: highlight-numbers@spacemacs-editing-visual... [51/112]
Package highlight-numbers is unavailable. Is the package name misspelled?
--> installing package: highlight-indentation@spacemacs-editing-visual... [52/112]
Package highlight-indentation is unavailable. Is the package name misspelled?
--> installing package: helm-xref@helm... [53/112]
Package helm-xref is unavailable. Is the package name misspelled?
--> installing package: helm-themes@helm... [54/112]
Package helm-themes is unavailable. Is the package name misspelled?
--> installing package: helm-swoop@helm... [55/112]
Package helm-swoop is unavailable. Is the package name misspelled?
--> installing package: helm-purpose@spacemacs-purpose... [56/112]
Package helm-purpose is unavailable. Is the package name misspelled?
--> installing package: helm-projectile@helm... [57/112]
Package helm-projectile is unavailable. Is the package name misspelled?
--> installing package: helm-org@helm... [58/112]
Package helm-org is unavailable. Is the package name misspelled?
--> installing package: helm-mode-manager@helm... [59/112]
Package helm-mode-manager is unavailable. Is the package name misspelled?
--> installing package: helm-make@helm... [60/112]
Package helm-make is unavailable. Is the package name misspelled?
--> installing package: helm-descbinds@helm... [61/112]
Package helm-descbinds is unavailable. Is the package name misspelled?
--> installing package: helm-ag@helm... [62/112]
Package helm-ag is unavailable. Is the package name misspelled?
--> installing package: helm@helm... [63/112]
Package helm is unavailable. Is the package name misspelled?
--> installing package: google-translate@spacemacs-language... [64/112]
Package google-translate is unavailable. Is the package name misspelled?
--> installing package: golden-ratio@spacemacs-navigation... [65/112]
Package golden-ratio is unavailable. Is the package name misspelled?
--> installing package: flycheck-package@emacs-lisp... [66/112]
Package flycheck-package is unavailable. Is the package name misspelled?
--> installing package: flycheck-elsa@emacs-lisp... [67/112]
Package flycheck-elsa is unavailable. Is the package name misspelled?
--> installing package: flx-ido@spacemacs-completion... [68/112]
Package flx-ido is unavailable. Is the package name misspelled?
--> installing package: fancy-battery@spacemacs-modeline... [69/112]
Package fancy-battery is unavailable. Is the package name misspelled?
--> installing package: eyebrowse@spacemacs-layouts... [70/112]
Package eyebrowse is unavailable. Is the package name misspelled?
--> installing package: expand-region@spacemacs-editing... [71/112]
Package expand-region is unavailable. Is the package name misspelled?
--> installing package: evil-visualstar@spacemacs-evil... [72/112]
Package evil-visualstar is unavailable. Is the package name misspelled?
--> installing package: evil-visual-mark-mode@spacemacs-evil... [73/112]
Package evil-visual-mark-mode is unavailable. Is the package name misspelled?
--> installing package: evil-tutor@spacemacs-evil... [74/112]
Package evil-tutor is unavailable. Is the package name misspelled?
--> installing package: evil-textobj-line@spacemacs-evil... [75/112]
Package evil-textobj-line is unavailable. Is the package name misspelled?
--> installing package: evil-surround@spacemacs-evil... [76/112]
Package evil-surround is unavailable. Is the package name misspelled?
--> installing package: evil-numbers@spacemacs-evil... [77/112]
Package evil-numbers is unavailable. Is the package name misspelled?
--> installing package: evil-nerd-commenter@spacemacs-evil... [78/112]
Package evil-nerd-commenter is unavailable. Is the package name misspelled?
--> installing package: evil-mc@multiple-cursors... [79/112]
Package evil-mc is unavailable. Is the package name misspelled?
--> installing package: evil-matchit@spacemacs-evil... [80/112]
Package evil-matchit is unavailable. Is the package name misspelled?
--> installing package: evil-lisp-state@spacemacs-evil... [81/112]
Package evil-lisp-state is unavailable. Is the package name misspelled?
--> installing package: evil-lion@spacemacs-evil... [82/112]
Package evil-lion is unavailable. Is the package name misspelled?
--> installing package: evil-indent-plus@spacemacs-evil... [83/112]
Package evil-indent-plus is unavailable. Is the package name misspelled?
--> installing package: evil-iedit-state@spacemacs-evil... [84/112]
Package evil-iedit-state is unavailable. Is the package name misspelled?
--> installing package: evil-goggles@spacemacs-evil... [85/112]
Package evil-goggles is unavailable. Is the package name misspelled?
--> installing package: evil-exchange@spacemacs-evil... [86/112]
Package evil-exchange is unavailable. Is the package name misspelled?
--> installing package: evil-easymotion@spacemacs-editing... [87/112]
Package evil-easymotion is unavailable. Is the package name misspelled?
--> installing package: evil-collection@spacemacs-evil... [88/112]
Package evil-collection is unavailable. Is the package name misspelled?
--> installing package: evil-cleverparens@spacemacs-evil... [89/112]
Package evil-cleverparens is unavailable. Is the package name misspelled?
--> installing package: evil-args@spacemacs-evil... [90/112]
Package evil-args is unavailable. Is the package name misspelled?
--> installing package: evil-anzu@spacemacs-evil... [91/112]
Package evil-anzu is unavailable. Is the package name misspelled?
--> installing package: eval-sexp-fu@spacemacs-editing... [92/112]
Package eval-sexp-fu is unavailable. Is the package name misspelled?
--> installing package: emr@emacs-lisp... [93/112]
Package emr is unavailable. Is the package name misspelled?
--> installing package: elisp-slime-nav@emacs-lisp... [94/112]
Package elisp-slime-nav is unavailable. Is the package name misspelled?
--> installing package: elisp-def@emacs-lisp... [95/112]
Package elisp-def is unavailable. Is the package name misspelled?
--> installing package: editorconfig@spacemacs-editing... [96/112]
Package editorconfig is unavailable. Is the package name misspelled?
--> installing package: dumb-jump@spacemacs-misc... [97/112]
Package dumb-jump is unavailable. Is the package name misspelled?
--> installing package: drag-stuff@spacemacs-editing... [98/112]
Package drag-stuff is unavailable. Is the package name misspelled?
--> installing package: dired-quick-sort@spacemacs-editing... [99/112]
Package dired-quick-sort is unavailable. Is the package name misspelled?
--> installing package: devdocs@spacemacs-misc... [100/112]
Package devdocs is unavailable. Is the package name misspelled?
--> installing package: define-word@spacemacs-language... [101/112]
Package define-word is unavailable. Is the package name misspelled?
--> installing package: column-enforce-mode@spacemacs-editing-visual... [102/112]
Package column-enforce-mode is unavailable. Is the package name misspelled?
--> installing package: clean-aindent-mode@spacemacs-editing... [103/112]
Package clean-aindent-mode is unavailable. Is the package name misspelled?
--> installing package: centered-cursor-mode@spacemacs-navigation... [104/112]
Package centered-cursor-mode is unavailable. Is the package name misspelled?
--> installing package: avy@spacemacs-editing... [105/112]
Package avy is unavailable. Is the package name misspelled?
--> installing package: auto-highlight-symbol@spacemacs-navigation... [106/112]
Package auto-highlight-symbol is unavailable. Is the package name misspelled?
--> installing package: auto-compile@emacs-lisp... [107/112]
Package auto-compile is unavailable. Is the package name misspelled?
--> installing package: all-the-icons@spacemacs-visual... [108/112]
Package all-the-icons is unavailable. Is the package name misspelled?
--> installing package: aggressive-indent@spacemacs-editing... [109/112]
Package aggressive-indent is unavailable. Is the package name misspelled?
--> installing package: ace-window@spacemacs-navigation... [110/112]
Package ace-window is unavailable. Is the package name misspelled?
--> installing package: ace-link@spacemacs-navigation... [111/112]
Package ace-link is unavailable. Is the package name misspelled?
--> installing package: ace-jump-helm-line@helm... [112/112]
Package ace-jump-helm-line is unavailable. Is the package name misspelled?
```
in the top window and
```
Debugger entered--Lisp error: (file-missing "Cannot open load file" "No such file or directory" "bind-map")
  require(bind-map)
  spacemacs-bootstrap/init-bind-map()
  funcall(spacemacs-bootstrap/init-bind-map)
  (let* ((pkg-name (eieio-oref pkg ':name)) (owner (car (eieio-oref pkg ':owners)))) (spacemacs-buffer/message (format "%S -> init (%S)..." pkg-name owner)) (funcall (intern (format "%S/init-%S" owner pkg-name))))
  configuration-layer//configure-package(#<cfgl-package cfgl-package-158d40dc5018>)
  mapc(configuration-layer//configure-package (#<cfgl-package cfgl-package-158d40bc8f48> #<cfgl-package cfgl-package-158d40dc503c> #<cfgl-package cfgl-package-158d40dc5018> #<cfgl-package cfgl-package-158d40bc8674> #<cfgl-package cfgl-package-158d40bc8688> #<cfgl-package cfgl-package-158d40bc86ac> #<cfgl-package cfgl-package-158d40bc86c0> #<cfgl-package cfgl-package-158d40bc86e4>))
  (let (packages-to-configure) (let ((tail packages)) (while tail (let ((pkg-name (car tail))) (let ((pkg (configuration-layer/get-package pkg-name))) (cond ((eieio-oref pkg ...) (spacemacs-buffer/message ...)) ((and ... ...) (spacemacs-buffer/message ...)) ((null ...) (spacemacs-buffer/message ...)) ((not ...) (spacemacs-buffer/message ...)) ((not ...) (spacemacs-buffer/message ...)) (t (let ... ...) (if ... nil ...) (cond ... ...)))) (setq tail (cdr tail))))) (setq packages-to-configure (reverse packages-to-configure)) (mapc 'configuration-layer//configure-package packages-to-configure) (mapc 'configuration-layer//post-configure-package packages-to-configure))
  configuration-layer//configure-packages-2((async bind-key bind-map diminish evil hydra use-package which-key))
  configuration-layer//configure-packages((abbrev ace-jump-helm-line ace-link ace-window aggressive-indent all-the-icons ansi-colors archive-mode async auto-compile auto-highlight-symbol avy bind-key bind-map bookmark buffer-menu centered-cursor-mode clean-aindent-mode column-enforce-mode compile conf-mode cus-edit dash debug default-helm-config default-ivy-config default-org-config define-word desktop devdocs diminish dired dired-quick-sort dired-x display-fill-column-indicator display-line-numbers doc-view doom-modeline dotenv-mode drag-stuff dumb-jump edebug ediff editorconfig eldoc electric-indent-mode elisp-def elisp-slime-nav emacs-lisp emr ...))
  configuration-layer//load()
  (cond (changed-since-last-dump-p (configuration-layer//load) (if (spacemacs/emacs-with-pdumper-set-p) (progn (configuration-layer/message "Layer list has changed since last dump.") (configuration-layer//dump-emacs)))) (spacemacs-force-dump (configuration-layer//load) (if (spacemacs/emacs-with-pdumper-set-p) (progn (configuration-layer/message (concat "--force-dump passed on the command line or configu..." "been reloaded, forcing a redump.")) (configuration-layer//dump-emacs)))) ((spacemacs-is-dumping-p) (configuration-layer//load)) ((and (spacemacs/emacs-with-pdumper-set-p) (spacemacs-run-from-dump-p)) (configuration-layer/message "Running from a dumped file. Skipping the loading p...")) (t (configuration-layer//load) (if (spacemacs/emacs-with-pdumper-set-p) (progn (configuration-layer/message (concat "Layer list has not changed since last time. " "Skipping dumping process!"))))))
  configuration-layer/load()
  (let ((please-do-not-disable-file-name-handler-alist nil)) (require 'core-spacemacs) (spacemacs/dump-restore-load-path) (configuration-layer/load-lock-file) (spacemacs/init) (configuration-layer/stable-elpa-init) (configuration-layer/load) (spacemacs-buffer/display-startup-note) (spacemacs/setup-startup-hook) (spacemacs/dump-eval-delayed-functions) (if (and dotspacemacs-enable-server (not (spacemacs-is-dumping-p))) (progn (require 'server) (if dotspacemacs-server-socket-dir (progn (setq server-socket-dir dotspacemacs-server-socket-dir))) (if (server-running-p) nil (message "Starting a server...") (server-start)))))
  (if (not (version<= spacemacs-emacs-min-version emacs-version)) (error (concat "Your version of Emacs (%s) is too old. " "Spacemacs requires Emacs version %s or above.") emacs-version spacemacs-emacs-min-version) (let ((please-do-not-disable-file-name-handler-alist nil)) (require 'core-spacemacs) (spacemacs/dump-restore-load-path) (configuration-layer/load-lock-file) (spacemacs/init) (configuration-layer/stable-elpa-init) (configuration-layer/load) (spacemacs-buffer/display-startup-note) (spacemacs/setup-startup-hook) (spacemacs/dump-eval-delayed-functions) (if (and dotspacemacs-enable-server (not (spacemacs-is-dumping-p))) (progn (require 'server) (if dotspacemacs-server-socket-dir (progn (setq server-socket-dir dotspacemacs-server-socket-dir))) (if (server-running-p) nil (message "Starting a server...") (server-start))))))
  eval-buffer(#<buffer  *load*> nil "/home/hove/.emacs.d/init.el" nil t)  ; Reading at buffer position 3366
  load-with-code-conversion("/home/hove/.emacs.d/init.el" "/home/hove/.emacs.d/init.el" t t)
  load("/home/hove/.emacs.d/init" noerror nomessage)
  startup--load-user-init-file(#f(compiled-function () #<bytecode -0x10ab10a44aba7b57>) #f(compiled-function () #<bytecode -0x1f3c61addc0a0c75>) t)
  command-line()
  normal-top-level()
```
and in the `*Backtrace*` window.

**Comment by tennysontbardwell** [comment]
Similar to or duplicate of https://github.com/syl20bnr/spacemacs/issues/16122

**Comment by joakim-hove** [comment]
> Similar to or duplicate of #16122

Sure I'll close this. I have become *very* dependent on spacemacs and would really to get back to a working spacemacs - right now I use plain vanilla emacs and am quite frustrated ... Please indicate if there is anything I can do to assist in debugging?

**Comment by joakim-hove** [comment]
Closed as duplicate of #16122
